### PR TITLE
Change 'or' to 'of' on line 9 of lost.md

### PR DESCRIPTION
**Pull request by wongbryan**
Fixed a typo in the latest blog post "lost.md"

### CODE REVIEW

File: posts/lost.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Lost in Thought
 
-Why is called *lost in thought*?
+Why is called _lost in thought_?
```
Comment (michaelnyu) [inline posts/lost.md:3]: Why is it called _lost in thought_?

**Review by michaelnyu — CHANGES_REQUESTED** [review]
thanks for this!  one change req

File: posts/lost.md (hunk 1 above)
Comment (wongbryan) [reply posts/lost.md:3]: Will have to bring this up with copywriters at the next standup. 

cc: Head of Copy @michaelnyu thoughts?

Comment (michaelnyu) [reply posts/lost.md:3]: looping in CCO (Chief Copywriter Officer) @wongbryan.  hope to get a decisions made before EOW thanks -

Comment (wongbryan) [reply posts/lost.md:3]: I'm for this change, will have to confirm with CFCP (Casca Fan Club President) @epicchewy. Thoughts on approving this?

Comment (epicchewy) [reply posts/lost.md:3]: Thank you for bringing this to my attention. This seems to be a large step in the right direction. It has my approval!

**Review by epicchewy — APPROVED** [review]
looks good to me

File: posts/lost.md (hunk 1 above)
Comment (wongbryan) [reply posts/lost.md:3]: Got it, making update. Good work boys.

**Review by wongbryan — COMMENTED** [review]
Updated to italics. Ready to merge!

**Comment by wongbryan** [comment]
bump

**Comment by michaelnyu** [comment]
thanks for this @wongbryan  !

**Comment by wongbryan** [comment]
> thanks for this @wongbryan !

No problem, be sure to leave me a good performance review this half 👍

**Pull request merged**